### PR TITLE
Docusaurus setup

### PR DESCRIPTION
**Pull request by TheBigBadBunce**
## Current state: http://8a3e3d89.ngrok.io/nestjs-admin/

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+const React = require('react')
```
Comment (TheBigBadBunce) [inline website/pages/en/index.js]: Tried using `import ... from` syntax but then the doc server refuses to start :(

File: docs/admin-entity.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
+---
+id: admin-entity
+title: The Admin Entity
```
Comment (williamdclt) [inline docs/admin-entity.md]:
```suggestion
title: Registering entities in the admin site
```

File: docs/admin-entity.md [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+sidebar_label: Admin Enitities
```
Comment (williamdclt) [inline docs/admin-entity.md]:
```suggestion
sidebar_label: Register entities
```

File: docs/admin-module.md [hunk 4]
```diff
@@ -1,4 +1,10 @@
-# Instantiating the AdminModule
+---
+id: admin-module
+title: The Admin Module
```
Comment (williamdclt) [inline docs/admin-module.md]:
```suggestion
title: The AdminModule
```

File: docs/admin-module.md [hunk 5]
```diff
@@ -1,4 +1,10 @@
...
+sidebar_label: Admin Modules
```
Comment (williamdclt) [inline docs/admin-module.md]:
```suggestion
sidebar_label: AdminModule
```

File: website/pages/en/index.js (hunk 1 above)
Comment (williamdclt) [reply website/pages/en/index.js]: 🤷‍♂️

File: website/siteConfig.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+// List of projects/orgs using your project for the users page.
+const users = []
+
+const siteConfig = {
+  title: 'NestJS-admin',
```
Comment (williamdclt) [inline website/siteConfig.js]:
```suggestion
  title: 'NestJS Admin',
```

File: website/static/css/custom.css [hunk 7]
```diff
@@ -0,0 +1,21 @@
+/* your custom css */
+
+.homeLogo {
+  width: 80%;
+  max-width: 900px;
```
Comment (williamdclt) [inline website/static/css/custom.css]: I think it was still a bit big?

**Pull request merged**